lib/internal/test_runner/runner.js

```diff
@@ -112,47 +121,131 @@ function getRunArgs({ path, inspectPort }) {
   return argv;
 }
 
+class FileTest extends Test {
+  #buffer = [];
+  #handleReportItem({ kind, node, level = 0, lexeme, tokens }) {
```

Comment:
```suggestion
  #handleReportItem({ kind, node, nesting = 0, lexeme, tokens }) {
```
I think this is a better name